eslint-custom-rules/jest-mock-path-exists.js

```diff
@@ -0,0 +1,110 @@
+const fs = require('fs')
+const path = require('path')
+
+const pathExistsCache = new Map()
+
+function pathExists(filePath) {
+  if (pathExistsCache.has(filePath)) {
+    return pathExistsCache.get(filePath)
+  }
+
+  try {
+    fs.statSync(filePath)
+    pathExistsCache.set(filePath, true)
+    return true
+  } catch (error) {
+    pathExistsCache.set(filePath, false)
+    return false
+  }
```

Comment:
on peut utiliser `existsSync` à la place d'un try / catch

on peut retourner la valeur get dans tout les cas

```suggestion
  if (!pathExistsCache.has(filePath)) {
  	pathExistsCache.set(filePath, fs.existsSync(filePath))
  }

  return pathExistsCache.get(filePath)
```